osf/migrations/0215_populate_registration_provider_notification_subscriptions.py

```diff
@@ -0,0 +1,73 @@
+import logging
+
+from django.db import migrations
+from django.contrib.auth.models import Group
+from guardian.shortcuts import assign_perm, get_perms, remove_perm
+
+logger = logging.getLogger(__file__)
+
+
+class RegistrationGroupHelper(object):
```

Comment:
Could we maybe pop out `GroupHelper` on `osf/migrations/0092_populate_notification_subscriptions.py` and subclass that here. It would be more concise and `GroupHelper` will likely have to be moved out of that file when that migration is squashed anyway.

Reply:
Do you have ideas on a specific place to pull GroupHelper out to? `osf.migrations.__init__` doesn't work well and leads to an `AppRegistryNotReady` error.

Reply:
Most likely this is happening because the model isn't being evaluated at the right time (like in the migration the pre-migration model is being loaded) so try `WhateverModel = apps.get_model('osf.<whatever_model>')` instread of importing and will ensure the app has loaded properly... I hope.

Reply:
Per conversation w/ Brian G, this has been changed to be a management command rather than a migration.